.github/workflows/commit_lint.yml

```diff
@@ -1,77 +0,0 @@
-name: "Conventional Commits"
-
-on:
-  pull_request:
-    types:
-      - opened
-      - edited
-      - reopened
-      - synchronize
-jobs:
-  main:
-    name: Validate commit messages
-    runs-on: ubuntu-latest
-    permissions:
-      pull-requests: write
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-          ref: ${{github.event.pull_request.head.ref}}
-          repository: ${{github.event.pull_request.head.repo.full_name}}
-
-      - name: Check commit message
-        id: check_commit_message
-        if: always()
-        run: |
-          set +e
-          
-          base_sha=${{ github.event.pull_request.base.sha }} 
-          head_sha=${{ github.event.pull_request.head.sha }}
-
-          output=$(.github/scripts/commit_check.sh "${base_sha}" "${head_sha}" 2>&1)
```

Comment:
Probably makes sense to also remove .github/scripts/commit_check.sh, since it's not going to be used anymore after this change.